var/spack/repos/spack_repo/builtin/packages/netcdf_c/package.py

```diff
@@ -139,6 +139,7 @@ class NetcdfC(CMakePackage, AutotoolsPackage):
     variant("zstd", default=True, description="Enable Zstandard compression plugin")
 
     depends_on("c", type="build")
+    depends_on("cxx", type="build")  # the cmake looks for c++ compiler
```

Comment:
@skosukhin Should we then add a:
```suggestion
    depends_on("cxx", type="build", when="buildsystem=cmake")
```
?